constitution/article-07.tex

```diff
@@ -7,7 +7,12 @@
 \Section{A-Board Status}
 \begin{enumerate}
   \item Members elected to be A-Board retain all standard Member rights, unless otherwise specified. 
-  \item Each individual on an A-Board position must directly report to those described in their respective sections. 
+  \item Each individual on an A-Board position must directly report to those described in their respective sections.
+  \item The creation of A-Board position is at the discretion of the E-Board.
```

Comment:
"\item The creation of an A-Board position is at the discretion of the E-Board."